packages/codefixes/src/typescript-codefix-collection.ts

```diff
@@ -86,45 +80,6 @@ export class TypescriptCodeFixCollection implements CodeFixCollection {
     };
   }
 
-  protected getFormatCodeSettingsForFile(filePath: string): FormatCodeSettings {
```

Comment:
Moved to helper to be called in fix plugins and pass settings to getCodefix* funstions

Reply:
This helps to deal with an async version of the prettier + function will be called once per file + all getCodefix functions can utilize settings when generating code changes.